pycqed/instrument_drivers/physical_instruments/DDM/DDMq.py

```diff
@@ -0,0 +1,318 @@
+'''
+    File:               DDM.py
+    Author:             Nikita Vodyagin, QuTech
+    Purpose:            control of Qutech DDM
+    Prerequisites:
+    Usage:
+    Bugs:
+'''
+
+
+from .SCPIddm import SCPIddm
+import numpy as np
+import struct
+from qcodes import validators as vals
+
+
+class DDMq(SCPIddm):
+
+    # def __init__(self, logging=True, simMode=False, paranoid=False):
+    def __init__(self, name, address, port, **kwargs):
+        super().__init__(name, address, port, **kwargs)
+        # AWG properties (to be overloaded by derived class)
+        self.device_descriptor = type('', (), {})()
```

Comment:
AFAIK these device descriptors and such are note used. 
Try commenting it out and testing if it still works. No sense in keeping artefacts of Wouters QWG driver.